region/maridia/inner-pink.json

```diff
@@ -837,8 +866,28 @@
                   "name": "Snail Climb",
                   "notable": false,
                   "requires": [
-                    "canSuitlessMaridia",
+                    "canSnailClimb",
+                    "h_canUseSpringBall",
+                    {"or": [
+                      "Gravity",
+                      {"and": [
+                        "canSuitlessMaridia",
+                        "HiJump"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Aqueduct Bootless Suitless Snail Climb (Right Door)",
```

Comment:
Looks like this ends up being strictly bootless. should there be a non-notable strat that uses gravity alone? (by adding `canUseEnemies`, or `canSnailClimb` to the `Base` strat, @ line 845, and 962)

Reply:
I didn't ever clarify anywhere that I was treating gravity as interchangeable with hijump.  
Gravity snail climb is a silly situation 🤣 But I did add a case here for jumping up with gravity and hijump here as a canUseEnemies.

Reply:
Should there be a gravity + snail climb strat for getting to 9?

Reply:
2 to 9 also has Gravity+Springball+canSnailClimb.
Gravity is a safer climb but I don't find it makes the trick much easier to execute, especially when the snail goes off screen.  And without underwater physics, the snail advances less in that situation.